lib/action_cable/channel/base.rb

```diff
@@ -183,6 +198,14 @@ def subscription_confirmation_sent?
           @subscription_confirmation_sent
         end
 
+        def reject!
```

Comment:
IMO, we should just use `#reject`. There's not a non-bang version of this method.